sonar-plugin-api/src/main/java/org/sonar/api/issue/Issue.java

```diff
@@ -106,6 +106,13 @@
   Integer line();
 
   /**
+   * @deprecated since 5.5, replaced by {@link #gap()}
+   */
+  @Deprecated
+  @CheckForNull
+  Double effortToFix();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)